code/modules/mob/living/simple_animal/hostile/handofgod.dm

```diff
@@ -0,0 +1,119 @@
+/mob/living/simple_animal/hostile/handofgod
+	name = "Hand of God"
+	desc = "A giant hand.  Smells like a great meal."
+	icon = 'icons/mob/handofgod.dmi'
+	icon_state = "handofgod"
+	icon_living = "handofgod"
+	icon_dead = "handofgod"
+	turns_per_move = 5
+	response_help_continuous = "rubs"
+	response_help_simple = "rub"
+	response_disarm_continuous = "nudges"
+	response_disarm_simple = "nudge"
+	speed = 0
+	maxHealth = 300
+	health = 300
+
+	harm_intent_damage = 0  //Mere mortal hands can't damage the visage of a god
+	obj_damage = 500  //He isn't fucking around
+	melee_damage_lower = 1
+	melee_damage_upper = 20  //He rolls the dice
+	attack_verb_continuous = "subjugates"
+	attack_verb_simple = "subjugate"
+	attack_sound = 'sound/magic/wandodeath.ogg'
+	speak_emote = list("deliciously declares")
+
+	//Hamburger helper helped her hamburger help her make a great meal
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 1500
+	faction = list("agoodmeal")
+	movement_type = FLYING
+	pressure_resistance = 200
+	del_on_death = TRUE
+	
+	var/peopleFed = 0
+	
+/mob/living/simple_animal/hostile/handofgod/Initialize()
+	. = ..()
+	var/datum/action/innate/handofgod/feed/F = new
+	F.Grant(src)
+	
+/datum/action/innate/handofgod
+	background_icon_state = "bg_alien"
+	
+/datum/action/innate/handofgod/feed
+	name = "Feed"
+
+/datum/action/innate/handofgod/feed/Activate()
+	var/mob/living/simple_animal/hostile/handofgod/H = owner
+	H.Feed()
+	
+/mob/living/simple_animal/hostile/handofgod/verb/Feed()
+	set category = "Hand of God"
+	set desc = "This will let you feed anyone nearby you.  They want your deliciousness"
+
+	var/list/choices = list()
+	for(var/mob/living/carbon/C in view(1,src))
+		if(Adjacent(C) && C.stat != DEAD)
+			choices += C
+
+	var/mob/living/M = input(src,"Who do you wish to feed?") in null|sortNames(choices)

```

Comment:
Use a pointed spell instead as introduced in #49909

Reply:
Spells, in my mob?  No way kiddo, I'm just gonna copy and paste spider code until it does what I want it to do